ruma-push-gateway-api/Cargo.toml

```diff
@@ -11,4 +11,11 @@ repository = "https://github.com/ruma/ruma"
 edition = "2018"
 
 [dependencies]
+js_int = { version = "0.1.9", features = ["serde"] }
 ruma-api = { version = "=0.17.0-alpha.1", path = "../ruma-api" }
+ruma-client-api = { version = "0.10.0-alpha.1", path = "../ruma-client-api" }
```

Comment:
The api crates should never depend on each other. If you need stuff from ruma-client-api here, it should probably be moved to ruma-common first.

Reply:
Sounds good. Extracted.